modules/monitoring/dashboard/dashboards/HPC.json.tpl

```diff
@@ -1,5 +1,6 @@
 {
   "displayName": "${title}: ${deployment_name}",
+  "labels": ${jsonencode(labels)},
```

Comment:
Did you do any testing around this. How confident are we that this won't break the created dashboard, and that the labels are applied as expected?

Reply:
I created this blueprint to test it:

```yaml
blueprint_name: test-dashboard-blueprint

vars:
  project_id:  ## Set GCP Project ID Here ##
  deployment_name: test-dashboard-deployment

deployment_groups:
- group: primary
  modules:
  - id: dashboard1
    source: modules/monitoring/dashboard
    settings:
      title: "My test dashboard"
      base_dashboard: HPC
      labels:
        label1: my_test_dashboard
        label2: my_other_label
```

Here's a screenshot of the created dashboard and the labels working as intended (note `label1` and `label2`):
![image](https://github.com/GoogleCloudPlatform/hpc-toolkit/assets/8553814/e03e9e9e-e83f-48e1-8d9c-acec1ec3e7d8)

Also just to be sure, I ran `gcloud monitoring dashboards describe "16158fb3-fa05-4cfc-9eae-ca6d8743d6d2"` and checked the `labels` property in the output:

```yaml
labels:
  ghpc_blueprint: test-dashboard-blueprint
  ghpc_deployment: test-dashboard-deployment
  ghpc_module: dashboard
  ghpc_role: monitoring
  label1: my_test_dashboard
  label2: my_other_label
```